### DISCUSSION TITLE
TP5 - Extra - División lenta III - Duda sobre la consigna

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#162] [Preguntas y respuestas] [answered]
Hola! en la consigna de division lenta III pide que **_Mantengan separado lo que ‘entra’ de lo que sale; dividendo y divisor por un lado, cociente y resto por otro._**. con mantener separado la salida de la entrada se refiere a usar dos estructuras diferentes?

**Comment by martinvilu** [comment, marked as answer]
> con mantener separado la salida de la entrada, ¿se refiere a usar dos estructuras diferentes?

Sí
